examples/index.js

```diff
@@ -150,19 +165,22 @@ let ciphertext
 let publicKey
 
 async function encrypt() {
-  console.log('Doing encryption')
-  setRequest('encrypt')
-
-  const c = await encryptWithPublicKey({
-    publicKey,
-    message: plaintext,
-  })
-  setResult(c)
-  console.log({ ciphertext: c })
-  ciphertext = c
+  // console.log('Doing encryption')
+  // setRequest('encrypt')
+  // const c = await encryptWithPublicKey({
+  //   publicKey,
+  //   message: plaintext,
+  // })
+  // setResult(c)
+  // console.log({ ciphertext: c })
+  // ciphertext = c
```

Comment:
Some commented out code here and the example `index.html`. Was it intentional?

Reply:
Yeah since encryption got removed, wasn't possible to encrypt from example.